src/gafaelfawr/providers/oidc.py

```diff
@@ -199,6 +211,51 @@ async def logout(self, session: State) -> None:
         """
         pass
 
+    async def _get_ldap_uidNumber(self, uid: str) -> int:
```

Comment:
We use Python naming conventions uniformly, so:
```suggestion
    async def _get_ldap_uid_number(self, uid: str) -> int:
```